app/presenters/menu/section.rb

```diff
@@ -16,6 +16,10 @@ def visible?
       auth
     end
 
+    def subsection?
```

Comment:
I tried to take the stuff from #6946 that can be merged w/o breaking anything here. So that we can move forward with the task...